README.md

```diff
@@ -1,2 +1,15 @@
 # community-protocols
-Cross-component coordination protocols
+As the diversity of custom elements grows wider and deeper, so does the benefits of those elements being able to communicate with each other via reusably well-known protocols that ensure components from X vendor can interoperate with components from Y vendor. This project looks to define community protocols for cross-component coordination that reduce the overhead required to make that possible without the side effect of locking consumers of those elements into specific patterns themselves.
```

Comment:
This paragraph is made up of two rather long sentences. They feel a but like run-ons. Perhaps break them up into shorter sentences. There are also some grammar errors, such as "so does the benefits" instead of something like "so does the benefit" or "so do the benefits".

Reply:
Ran this by grammarly, let me know what you think!